Add section Test Verifiable Credentials to VC "How it works" (#3055)
dfinity/portal: docs/developer-docs/identity/verifiable-credentials/how-it-works.mdx

For more details, check out the [specification in Internet Identity’s repository](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).

## Test verifiable credentials

There is a demo relying party deployed on ICP available for testing: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).

You first need to login with an identity provider. To avoid using real users, it is recommended to use the staging instance of Internet Identity: [https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/](https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/).

> There is a demo issuer deployed on ICP  available for testing. It will issue any requested credential. Feel free to use it when testing the flow: [qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).

Reviewer: ```suggestion
There is a demo issuer deployed on ICP available for testing. It will issue any requested credential. Feel free to use it when testing the flow: [qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
```

For more details, check out the [specification in Internet Identity’s repository](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).

> ## Test Verifiable Credentials

Reviewer: ```suggestion
## Test verifiable credentials
```

> There is a dummy relying party deployed in Internet Computer here: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).

Reviewer: ```suggestion
There is a demo relying party deployed on ICP available for testing: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).
```

> You first need to login with an Identity Provider. To avoid using real users, we recommend using the staging Internet Identity deployed here: [https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/](https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/).

Reviewer: ```suggestion
You first need to login with an identity provider. To avoid using real users, it is recommended to use the staging instance of Internet Identity: [https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/](https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/).
```

> We have deployed a dummy issuer that will issue any requested credential. Feel free to use it when testing the flow.
> 
> The dummy issuer is deployed in [canister id qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).

Reviewer: ```suggestion
There is a demo issuer deployed on ICP  available for testing. It will issue any requested credential. Feel free to use it when testing the flow: [qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
```
